contracts/ECRecovery.sol

```diff
@@ -3,12 +3,9 @@ pragma solidity ^0.4.23;
 
 /**
  * @title Eliptic curve signature operations
- *
```

Comment:
new lines between tags don't seem to hurt. But I'm ok removing them too.

Reply:
I'd prefer the new lines if there's no difference; makes the comment blocks much more readable.

Reply:
ah, just saw the above comment; let's remove them for now, yeah :(

Reply:
I'm happy either way. Your choice @shrugs.